.env

```diff
@@ -1,30 +1,23 @@
 local_state=./local_state
 
+TIME_ZONE=America/Chicago
+
 ####################
 # Needed for MIT-SHM, removing this should cause a performance hit see https://github.com/jessfraz/dockerfiles/issues/359
-# Use 'service:xorg' if running on a recent enough docker-compose version
-# Use 'host' if your Docker version can't support the service format
-SHARED_IPC=host
+SHARED_IPC=service:xorg
```

Comment:
This doesn't work when the `xorg` container is not running (aka: running on a desktop environment). I think it's better to leave `host` as the default since it'll work on all cases

Reply:
Good catch, I should have noticed that.  I'll fix it shortly.

Reply:
Do you know if there's any benefit to using the `service:xorg` variant on headless systems?  The (not written yet :wink:) `run-gow` script _could_ change this setting too, if there's any reason to.  I had a quick look at the docker docs and it seemed like it wouldn't really matter, though, and it'd be nice to keep it simpler if possible...

Reply:
I guess it's mostly a security feature; since that MIT-SHM is a way to share memory between processes it probably would be better to keep it separated from the host. But I really don't mind to leave it as `host` in a simple run script. 

Folks who are that concerned with security can always run things manually or in a different way..